### PR TITLE
gh-3 Return total order price in GetAllOrders REST API

### PR DESCRIPTION
**Pull request by Andrii77711**
Return total order price in GetAllOrders REST API https://github.com/Andrii77711/order-management-service/issues/3

### CODE REVIEW

File: src/main/java/com/to/ordermanagementservice/repository/impl/InMemoryOrderItemRepository.java [hunk 1]
```diff
@@ -16,16 +16,16 @@ public class InMemoryOrderItemRepository implements OrderItemRepository {
 
     public InMemoryOrderItemRepository() {
         OrderItem orderItem11 = new OrderItem();
-        orderItem11.setId(11);
+        orderItem11.setId(1);
```
Comment (taras-oglabyak) [inline src/main/java/com/to/ordermanagementservice/repository/impl/InMemoryOrderItemRepository.java]: please, set `id` here and in the rest of places to prev. values

File: src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java [hunk 2]
```diff
@@ -43,8 +44,20 @@ private OrderDetails collectOrderDetails(Order order) {
...
+
+    private BigDecimal getTotalPriceForOrder(List<OrderItemDetails> orderItems){
+        BigDecimal result = new BigDecimal(0);
+        for (OrderItemDetails orderItem:orderItems){
+            BigDecimal totalPrice = new BigDecimal(orderItem.getQuantity());
+            totalPrice = totalPrice.multiply(orderItem.getPrice());
+            result = result.add(totalPrice);
+        }
+        return result;
+    }
```
Comment (taras-oglabyak) [inline src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java]: please, check code formatting

**Review by taras-oglabyak — CHANGES_REQUESTED** [review]

File: src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java [hunk 3]
```diff
@@ -43,8 +44,20 @@ private OrderDetails collectOrderDetails(Order order) {
...
     }
+
+    private BigDecimal getTotalPriceForOrder(List<OrderItemDetails> orderItems){
+        BigDecimal result = new BigDecimal(0);
+        for (OrderItemDetails orderItem:orderItems){
+            BigDecimal totalPrice = new BigDecimal(orderItem.getQuantity());
+            totalPrice = totalPrice.multiply(orderItem.getPrice());
+            result = result.add(totalPrice);
+        }
+        return result;
```
Comment (taras-oglabyak) [inline src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java]: Alternative code may be 
```java
BigInteger result = BigInteger.ZERO;
for (...) {
    BigInteger itemTotal = orderItem.getPrice().multiply(BigInteger.valueOf(orderItem.getQuantity()));
    result = result.add(itemTotal);
}
return result;

File: src/main/java/com/to/ordermanagementservice/repository/impl/InMemoryOrderItemRepository.java (hunk 1 above)
Comment (Andrii77711) [reply src/main/java/com/to/ordermanagementservice/repository/impl/InMemoryOrderItemRepository.java]: changed value

File: src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java (hunk 2 above)
Comment (Andrii77711) [reply src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java]: fixed

**Review by taras-oglabyak — APPROVED** [review]

File: src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java (hunk 3 above)
Comment (Andrii77711) [reply src/main/java/com/to/ordermanagementservice/service/impl/SimpleOrderService.java]: do it

**Review by taras-oglabyak — APPROVED** [review]

**Pull request merged**